lib/config/executor/container_vm.go

```diff
@@ -141,6 +141,9 @@ type ExecutorConfig struct {
 	// TODO: a bit docker specific
 	RepoName string `vic:"0.1" scope:"read-only" key:"repo"`
 
+	// Resources specifies the CPU count and memory in a container VM
+	Resources Resources
```

Comment:
This should not be here. https://github.com/vmware/vic/blob/HEAD/lib/spec/spec.go#L57 instead.